WooCommerce/Classes/ViewModels/Order Details/OrderDetailsPaymentAlerts.swift

```diff
@@ -77,8 +77,8 @@ final class OrderDetailsPaymentAlerts {
         presentViewModel(viewModel: viewModel)
     }
 
-    func error(error: Error, tryAgain: @escaping () -> Void) {
-        let viewModel = errorViewModel(error: error, tryAgain: tryAgain)
+    func error(error: Error, tryAgain: @escaping () -> Void, dismissError: @escaping (UIViewController?) -> Void) {
```

Comment:
What do you think about an alternative option here: make the passed in `dismissError` a dismiss completion handler instead. Then set the secondary action of the view model to a closure that calls `viewController.dismiss` with the `onCompletion` that was passed in.

That way you don't have to pass the view controller to the closure, nor handle the dismiss itself in several places.

Reply:
Great suggestion! Updated in https://github.com/woocommerce/woocommerce-ios/pull/6639/commits/9380bed2e961425d9ba815ef57b0dc92643fe127